client/src/components/Containers/FirstContainer.js

```diff
@@ -1,33 +1,33 @@
 import React from "react";
-import { Container, Grid, Box, Typography } from "@material-ui/core";
+import { Grid, Typography } from "@material-ui/core";
 import useStyles from "./FirstContainerStyles";
 import MacBook from "../../assets/images/MacBook.svg";
 
 function FirstContainer() {
   const classes = useStyles(useStyles);
 
   return (
-    <Container
-      className={classes.first_container}
-      maxWidth={true}
-      component="div"
-    >
-      <Grid container justify="center" alignItems="stretch" direction="column">
-        <Box className={classes.box_in_first_container}>
-          <Typography className={classes.root} variant="h2">
+      <Grid className={classes.first_container} container alignItems="center" spacing={5} direction="row">
```

Comment:
I have deleted `spacing={5}` to avoid overflow. More info [here](https://stackoverflow.com/questions/45519275/grid-in-material-ui-causes-horizontal-scroll-react).